tests/FSharp.Compiler.ComponentTests/Interop/DeeplyNestedCSharpClasses.fs

```diff
@@ -47,21 +47,21 @@ let loss2 = MyNamespace.OoterClass.InnerClass.MoreInnerClass.somefunction()   //
 
         let fsharpSource =
             """
-let loss2 = MyNamespace.OuterClass.InerClass.MoreInnerClass.somefunction()   //Note the miss-typed InnerClass name we expect a good error message
+let loss2 = MyNamespace.OuterClass.InnerClass_.MoreInnerClass.somefunction()   //Note the mistyped InnerClass name we expect a good error message
```

Comment:
InnerClass probably should also work.

Reply:
No, this is the case where InnerClass is the intentionally misspelled thing (see the comment)

Reply:
Maybe can keep InerClass then